packages/webdoc-default-template/tmpl/document.tmpl

```diff
@@ -56,5 +56,10 @@ docList.forEach((doc) => {
       type: ["MethodDoc", "FunctionDoc"]
     }
   }) ?>
+
+  <hr id="nav-members-separator" />
```

Comment:
Remove `hr` tags, this should be done on `css` side, to make template more flexible for editing.
It's easier to edit how template *looks* from `css` without making modifications into template itself.